Pipfile

```diff
@@ -56,8 +56,8 @@ raven = "==6.9.0"
 requests = ">=1.2.0"
 social-auth-app-django = ">=2.0.0"
 text-unidecode = "*"
-uwsgi = {version = ">=2.0.0", platform_system = "!= 'Windows'"}
-celery = {version = "*", extras = ["redis"]}
+uwsgi = {version = ">=2.0.0",platform_system = "!= 'Windows'"}
```

Comment:
It looks like the official docs is using spaces as in https://pipenv.readthedocs.io/en/latest/advanced/#specifying-basically-anything, could we add the spaces back? It has cleaner styles.

Reply:
Change was done by pipenv, did not modified manually. (pipenv, version 2018.11.26)

Reply:
Oh never mind, it is bug with pipenv and is fixed in master. It can be fixed manually by removing spaces though. I think it's fine since it's so minor.